src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -547,7 +553,19 @@ internal Result CreateResult(ResultVersionOne v1Result)
 
                 if (!string.IsNullOrWhiteSpace(v1Result.Snippet))
                 {
-                    result.SetProperty($"{FromPropertyBagPrefix}/snippet", v1Result.Snippet);
+                    Region region = result.Locations?[0]?.PhysicalLocation?.Region;
+
+                    if (region != null)
+                    {
+                        region.Snippet = new FileContent
+                        {
+                            Text = v1Result.Snippet
+                        };
+
+                        region.Length = v1Result.Snippet.Length;
+                        region.StartColumn = 1;
+                        region.EndColumn = region.Length + 1;
+                    }
```

Comment:
I'm sorry, I was unclear. I meant: "For the sake of the people who read the v1 sample file, make the region properties in the v1 sample file consistent with the snippet in the v1 sample file." When it comes to actually _copying_ the properties from v1 to v2, just take what they give you. If the v1 file was inconsistent, tough luck. (In fact, you _have_ to just copy the properties, for the sake of round-tripping (unless and until we adopt the "stuff the whole v1 run in the v2 run's property bag" trick).

Reply:
ok sorry, should have known that :p